alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -117,6 +117,8 @@ class AlertIndices(
         /** The in progress alert history index. */
         const val ALERT_INDEX = ".opendistro-alerting-alerts"
 
+        const val ALERT_CONFIG_INDEX = ".opendistro-alerting-config"
```

Comment:
I added this here because I see it in the [security repo](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java#L62), but I'm not able to find references to it in this repo. Where is this index declared and how is it used in the alerting plugin?

Reply:
https://github.com/opensearch-project/common-utils/blob/889bdb8eb47b334eb1fb8c0e20465f0934403af0/src/main/kotlin/org/opensearch/commons/alerting/model/ScheduledJob.kt#L17

Reply:
Thanks @bowenlan-amzn! I updated the PR to refer to this constant.